src/spetlr/delta/delta_handle.py

```diff
@@ -25,22 +28,49 @@ class DeltaHandleInvalidFormat(DeltaHandleException):
 
 
 class DeltaHandle(TableHandle):
-    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+    def __init__(
+        self,
+        name: str,
+        location: str = None,
+        data_format: str = "delta",
+        options_dict: dict = None,
+        ignore_changes: bool = True,
+        stream_start: datetime = None,
+        max_bytes_per_trigger: int = None,
+    ):
+        """ """
         self._name = name
         self._location = location
         self._data_format = data_format
 
         self._partitioning: Optional[List[str]] = None
-
         self._validate()
 
+        self._options_dict = (
```

Comment:
since these are all things that may want to be changed after creation (setting them from the configurator is not possible), please add a `setter` method and then call that setter method from the init method. let the setter mehtod handle the setting and overwriting of values

Reply:
Implemented